en/04_Changelogs/5.0.0.md

```diff
@@ -1711,9 +1724,22 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Changed return type for [`Symbiote\QueuedJobs\Services\QueuedJobHandler::write()`](api:Symbiote\QueuedJobs\Services\QueuedJobHandler::write()) from dynamic to `void`
 - Changed parameter type in [`Symbiote\QueuedJobs\Services\QueuedJobHandler::write()`](api:Symbiote\QueuedJobs\Services\QueuedJobHandler::write()) for `$record` from `array` to `Monolog\LogRecord`
 
-#### tractorcow/silverstripe-fluent
+### tractorcow/silverstripe-fluent
 
 - Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::getLinkingMode()` - use `LocaleInformation()` instead
 - Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::LocaleLink()` - use `LocaleInformation` instead
 
+</details>
+
+## Full commits list
+
+This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
```

Comment:
This probably needs to be reworded now that it's not in the bug fixes section - this wording has a heavy emphasis on bug fixes, but the commits list isn't _just_ about bug fixes.